config/config_loader.go

```diff
@@ -138,6 +140,17 @@ func loadConsumerConfig() {
 		ref.Implement(rpcService)
 	}
 
+	// Write current configuration to cache file.
+	if consumerConfig.CacheFile != "" {
+		if data, err := yaml.MarshalYML(consumerConfig); err != nil {
+			logger.Errorf("Marshal consumer config err: %s", err.Error())
+		} else {
+			if err := ioutil.WriteFile(consumerConfig.CacheFile, data, 0666); err != nil {
```

Comment:
I can not get this point. why will you copy this config to a new file?

Reply:
Please see the issue comment: https://github.com/apache/dubbo-go/issues/729#issuecomment-681617545.